### PR TITLE
Test and fix raw_cmd() on MySQL

### PR DESCRIPTION
**Pull request by tomhoule**
closes https://github.com/prisma/quaint/issues/187

### CODE REVIEW

File: src/connector/mysql.rs
Comment (pimeys) [inline src/connector/mysql.rs:419]: temporary table?

Comment (tomhoule) [reply src/connector/mysql.rs:419]: This is crashing. Let's discuss this later, but I'm getting pessimistic about fixing this.

File: src/connector/mysql.rs
Comment (tomhoule) [inline src/connector/mysql.rs:190]: note: run `cargo fmt` manually. I think it doesn't work automatically on my setup because of the feature flags.

File: src/connector/mysql.rs
Comment (pimeys) [inline src/connector/mysql.rs:405]: This could also be `test_all_connector` in `query.rs`.

**Review by pimeys — CHANGES_REQUESTED** [review]
Maybe extend the tests to all databases, code is good.

File: src/tests/query/error.rs
Comment (pimeys) [inline src/tests/query/error.rs:300]: Maybe use temporary tables here?

File: src/tests/query/error.rs
Comment (tomhoule) [inline src/tests/query/error.rs:296]: This test is challenging to make cross-database, the create table abstraction in `TestApi` doesn't really work because we specifically want to test multiple statements in the same script. So it would have a bunch of database-specific code, or a new create table abstraction.

Comment (pimeys) [reply src/tests/query/error.rs:296]: Something similar to this? https://github.com/prisma/quaint/blob/master/src/tests/test_api/mysql.rs#L43-L56

Comment (tomhoule) [reply src/tests/query/error.rs:296]: It wouldn't test what we want to test, because we want the 4 statements in one call.

Comment (pimeys) [reply src/tests/query/error.rs:296]: We could have an additional function to that...

**Comment by tomhoule** [comment]
@pimeys So let's maybe wait until next release is done to merge this.

**Comment by tomhoule** [comment]
@pimeys, from our last discussions I'm going to assume it's ok to merge this now, so we can get the fix down the pipeline and test it as soon as possible.

**Comment by pimeys** [comment]
Gogogo.

**Pull request merged**
